tests/test_androidmodule_ctypes_finder.py

```diff
@@ -1,6 +1,12 @@
 
-import mock
-from mock import MagicMock
+# This test is still expected to support Python 2, as it tests
+# on-Android functionality that we still maintain
+try:  # Python 3+
+    from unittest import mock
+    from unittest.mock import MagicMock
+except ImportError:  # Python 2
+    import mock
+    from mock import MagicMock
```

Comment:
It isn't installed on my desktop :p 

I decided I preferred to be explicit about the Python 2 exception for the limited subset of files where we still need it. I'm willing to change it if you think it's better, but I consider this innocuous.

Reply:
It doesn't need to be on your desktop since it's in the `tox.ini`.
It's definitely minor/preference so if you prefer explicit that's fine for me :+1:

Reply:
I would prefer keeping it in a way that works with the standard library (`from unittest import mock`). I personally don't mind these python 2 special branches because it reminds us that it needs to go :smile: if we change this to `import mock` nobody will remember to change it back, and I definitely think in the long term doing the right standard library import is better

Reply:
Thanks for sharing. I actually don't mind that much. I'm just being too lazy and I prefer one line over 4. Specially since it has to be repeated over a lot of (yet to come) tests files